conduct.md

```diff
@@ -7,16 +7,268 @@
 
 ## Categories of Conduct
 
-* Attendance
-* Deadlines
-* Communication
-* Participation
-* Conduct
-* Decisions
-* Postmortems
-* Conflicts
-* Infractions
+* [Conduct](#conduct)
+* [Attendance](#attendance)
+* [Deadlines](#deadlines)
+* [Communication](#communication)
+* [Participation](#participation)
+* [Decisions](#decisions)
+* [Postmortems](#postmortems)
+* [Conflicts](#conflicts)
+* [Infractions](#infractions)
+
+## Conduct
+
+In this context, we define conduct as the way a team member behaves and the way
+they act when carrying out their part of the project. First, everyone
+is obligated to adhere to the nine categories in the Code of Conduct. They will be
+our foundation throughout the semester when working together. Moving on, everyone
+should respect one another; no one should be excluded based on their race or
+gender. Team members should not shun the ideas of anyone, especially those who
+may have not have had an equal amount of experience as others. Also, at the
+beginning of each session, everyone should enter with positive attitudes, these
+positive attitudes should stay consistent whether the project is working or
+breaking.
+
+* **Acceptable Conduct (C):**
+  * *Members shall*
```

Comment:
This document uses "Members shall", "Members shall not", "Members Shall", "No Member Shall", "Team Members shall NOT", and some others for team leads.  These should be changed to always be consistent with wording and capitalization.